targets/targets.json

```diff
@@ -2881,13 +2881,16 @@
         "inherits": ["ARM_IOTSS_Target"],
         "core": "Cortex-M3",
         "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
-        "extra_labels": ["ARM_SSG", "CM3DS_MPS2"],
+        "extra_labels": ["ARM_SSG", "CM3DS_MPS2", "SMSC9220"],
```

Comment:
If using the component directory (which I've confirmed is correct thing to do), you should have this instead as

    "components": ["SMSC9220"],